packages/server/src/ChatflowPool.ts

```diff
@@ -17,8 +17,19 @@ export class ChatflowPool {
      * @param {ICommonObject} overrideConfig
      */
     add(chatflowid: string, endingNodeData: INodeData | undefined, startingNodes: IReactFlowNode[], overrideConfig?: ICommonObject) {
+        const removePrefix = (colName: string) => colName.replace(/^"[^"]*"/, '') // Regex to remove prefix
```

Comment:
thanks for the suggested fix!

why do we have the fix here though? I thought it should be in the `CSVAgent.ts` line 178 to check if the `pythonCode` contains any prefix

Reply:
I thought of changing it directly from the Pool because not only CSV AGENT AirtableAgent also had same issue, but yes I do find changing from CSV AGENT is precise, will revert back and do he checks in both CSV and AirtableAgent

Reply:
Any news about this fix?